frameworks_test.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/kudobuilder/kudo/pkg/test"
+)
+
+// Launch the KUDO tests.
+// To add more tests, add them to TestDirs belowe.
+func TestKudoFrameworks(t *testing.T) {
+	harness := test.Harness{
+		T:         t,
```

Comment:
can we add `goimports` check from the start? 

```
 -		T:         t,
 -		TestDirs:  []string{
 +		T: t,
 +		TestDirs: []string{
  			"./repository/zookeeper/tests/",
  		},
```

`goimports` would fail on this line, just detected this in CI builds where we run `goimports`

Reply:
I'd prefer not to spend time on that, as this isn't a Go repository, we'll be dropping this in favor of running the harness with kudoctl: kudobuilder/kudo#352